src/net/java/sip/communicator/plugin/desktoputil/plaf/SearchFieldUI.java

```diff
@@ -177,10 +179,21 @@ protected void customPaintBackground(Graphics g)
                 - searchIcon.getIconHeight()/2;
 
             g2.drawImage(searchIcon.getImage(), 3, dy, null);
+            String str = c.getText();
+            boolean containsLetter = false;
+
+            for (int i = 0; i < str.length(); i++)
+            {
+                if (Character.isLetter(str.charAt(i)))
```

Comment:
Possible NPE. Please, THINK and ANALYZE before doing changes.

Use isAlpha here as well. I doube the code is used at all anyway.

Reply:
Right. Sorry. No rush anymore. FIxed